COPYING.txt

```diff
@@ -4,4 +4,4 @@ http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
 Please see LICENSE.txt for the full text of the OSL 3.0 license or contact license@magentocommerce.com for a copy.
 
 Subject to Licensee's payment of fees and compliance with the terms and conditions of the MEE License, the MEE License supersedes the OSL 3.0 license for each source file.
-Please see <insert file name of the MEE license> for the full text of the MEE License or visit http://magento.com/legal/terms/enterprise.
\ No newline at end of file
+Please see LICENSE_EE.txt for the full text of the MEE License or visit http://magento.com/legal/terms/enterprise.
```

Comment:
I am personally uncomfortable adding the LICENSE which as of now is only available in software that is supposed to be kept private to the repository.

I would not want to open myself nor my employer up to any sort of legal issue related to revealing things we shouldn't.